### PR TITLE
Initial structure for notion

### PR DESCRIPTION
**Pull request by SamWilsn**
So this is my stab at #1 and to create a basic structure for the project as a whole.

I've assumed a few things here, just to get the project started. Please voice any concerns or displeasure!

# Things to Talk About

 - The name. Do we like it? Would muslin be better?
 - This pull request configures [bors](https://bors.tech). bors is a bot that enables a particular workflow on top of a normal CI platform and Github pull requests.
 - This pull request configures [Travis CI](https://travis-ci.com/) with [clippy](https://github.com/rust-lang/rust-clippy) and [rustfmt](https://github.com/rust-lang/rustfmt) enabled. These tools will enforce "good" Rust patterns and code format respectively.
 - The client and server crates are libraries, with a binary in `src/bin/`. This is to enable better testing without having to go through the CLI. Not necessarily a requirement, but if we want to go heavy on testing, it might be nice.

# Merging

To merge with bors, comment `bors r+`.

### CODE REVIEW

File: toile-cli/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[package]
+name = "toile-cli"
+version = "0.1.0"
+authors = ["Sam Wilson <sam.wilson@consensys.net>"]
```
Comment (gjtrowbridge) [inline toile-cli/Cargo.toml]: If this is meant to be the repo where we store all the combined work, probably makes sense to add me and Ansgar to all these toml files also.

File: toile-server/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[package]
+name = "toile-server"
+version = "0.1.0"
+authors = ["Sam Wilson <sam.wilson@consensys.net>"]
```
Comment (gjtrowbridge) [inline toile-server/Cargo.toml]: Same as above

**Review by gjtrowbridge — COMMENTED** [review]
> So this is my stab at #1 and to create a basic structure for the project as a whole.
> 
> I've assumed a few things here, just to get the project started. Please voice any concerns or displeasure!
> # Things to Talk About
> 
>     * The name. Do we like it? Would muslin be better?
> 
>     * This pull request configures [bors](https://bors.tech). bors is a bot that enables a particular workflow on top of a normal CI platform and Github pull requests.
> 
>     * This pull request configures [Travis CI](https://travis-ci.com/) with [clippy](https://github.com/rust-lang/rust-clippy) and [rustfmt](https://github.com/rust-lang/rustfmt) enabled. These tools will enforce "good" Rust patterns and code format respectively.
> 
>     * The client and server crates are libraries, with a binary in `src/bin/`. This is to enable better testing without having to go through the CLI. Not necessarily a requirement, but if we want to go heavy on testing, it might be nice.
> 
> 
> # Merging
> 
> To merge with bors, comment `bors r+`.

Thanks for putting this together Sam!  Some thoughts:

- No opinion on Bors here, but would be nice if it wasn't obligatory to use it (not sure I want to add learning a new tool immediately, even a simple one, on top of Rust + all the other Eth concepts).
- Love the Rustfmt / Travis stuff

**Review by gjtrowbridge — APPROVED** [review]
Meant to approve earlier -- only changes requested are the names in the toml files, but those are easy to change later also.  Thanks again for putting this together.

File: toile-cli/Cargo.toml (hunk 1 above)
Comment (SamWilsn) [reply toile-cli/Cargo.toml]: What email addresses would you and @adietrichs like me to include?

Comment (gjtrowbridge) [reply toile-cli/Cargo.toml]: Just my consensys one works, assuming same for Ansgar?  Mine is gregory.trowbridge@consensys

**Comment by SamWilsn** [comment]
bors try

**Comment by SamWilsn** [comment]
bors try

**Pull request merged**